src/repositories/searchResult.ts

```diff
@@ -19,13 +19,16 @@ export type SearchResultScraping = Prisma.SearchResultGetPayload<{
   select: typeof searchResultScraping;
 }>;
 
-export const createSearchResult = async (
+export const createMultipleSearchResults = async (
```

Comment:
How about being consistent with Prisma naming:

```suggestion
export const createManySearchResults = async (
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/78/commits/bd13796cf9665b67851e198390d976e544820185